.github/workflows/netlify.yml

```
@@ -1,11 +1,11 @@
-name: Re-run build from 'main' on Netlify every two hours
+name: Re-run build from 'main' on Netlify every six hours
 on:
   schedule:
     - cron: '0 */6 * * *'
   pull_request:
```

```suggestion
```

Do not think this is actually required?

You may want to change it to

```suggestion
  workflow_dispatch:
```

so that it can be triggered from the GitHub UI.